### ISSUE TITLE
How to run postCreateCommand in another folder?

### ISSUE THREAD

**Issue by DanTup** [microsoft/vscode-remote-release#349]
Currently it seems like `postCreateCommand` always runs in the folder that was opened, but in the example containers the projects are all in subfolders named `test-project`. I tried doing this:

```js
"postCreateCommand": ["cd test-project", "pub get"]
```

But it failed like this:

```
OCI runtime exec failed: exec failed: container_linux.go:344: starting container process caused "exec: \"cd test-project\": executable file not found in $PATH": unknown
postCreateCommand ["cd test-project","pub get"] failed.
```

Is there some way to do this? If not, would this be considered?

**Comment by Chuxel** [comment]
@DanTup You should be able to do this using a string as follows: `"postCreateCommand": ["cd test-project", "pub get"]`

The array is for separated arguments of a single command.  e.g. `"postCreateCommand": ["cd", "test-project"]`

We can clarify this better in docs looking at it.

**Comment by Chuxel** [comment]
Whoops!  Cmd+Z'd myself into a bad example.  Meant: `"postCreateCommand": "cd test-project && pub get"`

**Comment by DanTup** [comment]
I tried that but it failed with the same message.

As it turned out, it probably wasn't a good place to do `pub get`, since this is project-specific, not container-specific. I was really trying it as a workaround until I get to fix https://github.com/Dart-Code/Dart-Code/issues/1719 (the test-project folder being in a subfolder would prevent us noticing it needed packages fetching). With the simplified Dart container there aren't any dependencies now, so I was able to avoid it.

Though now I'm wondering whether `postCreateCommand` differs much from just putting an extra `RUN` section at the bottom of the dockerfile?
